quilt/tools/command.py

```diff
@@ -160,6 +161,16 @@ def _parse_package(name):
 
     return owner, pkg
 
+def _open_url(url):
+    if sys.platform == 'win32':
+        os.startfile(url)
+    elif sys.platform == 'darwin':
+        devnull = os.open(os.devnull, os.O_RDWR)
+        subprocess.call(['open', url], stdin=devnull, stdout=devnull, stderr=devnull)
+    else:
+        devnull = os.open(os.devnull, os.O_RDWR)
+        subprocess.call(['xdg-open', url], stdin=devnull, stdout=devnull, stderr=devnull)
+
```

Comment:
* to be safe we should a should add a unit test to try the above calls in CI so we have advance notice if something breaks
* also do we need `try/catch` in `command.py`? otherwise this will throw to the top.